workers/unity/Packages/io.improbable.gdk.core/Systems/EntitySystem.cs

```diff
@@ -1,61 +1,124 @@
-using System.Collections.Generic;
+using System;
+using Unity.Collections;
 using Unity.Entities;
-using Unity.Profiling;
+using Unity.Jobs;
 
 namespace Improbable.Gdk.Core
 {
     [DisableAutoCreation]
     [AlwaysUpdateSystem]
     [UpdateInGroup(typeof(SpatialOSReceiveGroup.InternalSpatialOSReceiveGroup))]
-    [UpdateBefore(typeof(SpatialOSReceiveSystem))]
-    public class EntitySystem : ComponentSystem
+    [UpdateAfter(typeof(SpatialOSReceiveSystem))]
+    public class EntitySystem : SystemBase
```

Comment:
I don't think we need this system. The only usages are in the WorkerInspector and Subscription. Both of which can use `WorkerSystem.Diff.GetEntitiesRemoved()` or `WorkerSystem.Diff.GetEntitiesAdded()`. 

Thoughts?